installer/roles/ansible-service-broker-setup/files/provision-ansible-service-broker.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
The openshift-ansible repo already has a role for this which we could package and contribute to WDYT? https://github.com/openshift/openshift-ansible/tree/882241d56dcfc2654f49ea3296ccb8e3b118f8b8/roles/ansible_service_broker